Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -243,7 +245,12 @@ public override void Update()
 			     player.stats.CanUseStat( Stat.Stamina ) &&
 			     player.physics.ClimbCheck() )
 			{
-				player.physics.ClimbSurface( player.controls.GetMoveInput() );
+				Vector3 moveInput = player.controls.GetMoveInput();
+				player.physics.ClimbSurface( moveInput );
+				player.animator.SetFloat( "moveSpeed", moveInput.magnitude );

```

Comment:
`PlayerPhysics.normalizedMoveSpeed` isn't correct when climbing because it zeroes-out the Y component of the movement vector (essentially getting the horizontal move speed). In this case I want a value between 0 and 1, which using the input gets me.